### PR TITLE
[#859] Add google rich card carousel to render library

### PR DESCRIPTION
**Pull request by thorstenairy**

### CODE REVIEW

File: lib/typescript/render/components/RichCardCarousel/index.tsx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  const {cardContents, cardWidth, id, fromContact, inChatPlugin} = props;
+  const [position, setPosition] = useState(0);
+  const [disabled, setDiabled] = useState(false);
+  const amountCards = cardContents.length;
+
+  const button = (position: number, amountCards: number, id: string) => {
+    if (position == 0) {
+      return (
+        <button
+          className={styles.moveNext}
```
Comment (bitboxer) [inline lib/typescript/render/components/RichCardCarousel/index.tsx:70]: if a button is not a submit in a form, you should always add a `type='button'` to it.  The default type is submit sadly.

Comment (bitboxer) [reply lib/typescript/render/components/RichCardCarousel/index.tsx:70]: Same with the other buttons below

File: lib/typescript/render/components/RichCardCarousel/index.tsx [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+                </div>
+              </div>
+            </div>
+          </div>
+        </>
+      ) : (
+        <>
+          <div
+            className={styles.containerMember}
+            style={inChatPlugin == true ? {transform: 'scale(0.7)'} : {transform: 'scale(1.0)'}}>
```
Comment (bitboxer) [inline lib/typescript/render/components/RichCardCarousel/index.tsx:165]: Is the `scale(1.0)` really needed?

**Review by bitboxer — DISMISSED** [review]
Really awesome stuff, just some minor nitpicks

**Review by chrismatix — APPROVED** [review]

**Pull request merged**